andela/helios-ah-backend
# Fix article like bug

#### What does this PR do?
This PR allows a user to edit like status accordingly and repeatedly

#### Description of Task to be completed?
* defaults the like status to 'false' when edited (`put`)
* when a parameter (`isLiked`) is supplied it overrides the default status

#### How should this be manually tested?
* clone the repository  ` https://github.com/andela/helios-ah-backend.git `
* cd to the  ` helios-ah-backend ` folder
* check out the `bg-fix-article-like-bug` branch
* run `npm i` on the bash terminal
* run `npm run test` on the terminal

#### What are the relevant pivotal tracker stories?
#161779785, #161779787

#### Any background context you want to add?
N/A
